ipmi/Sensor Discovery/doc/README.head.md

```diff
@@ -18,9 +18,19 @@ required by most of the [IPMI templates](../../).
    script into your Zabbix servers/proxy external script directory
    ([`ExternalScripts`](https://www.zabbix.com/documentation/3.0/manual/appendix/config/zabbix_server)).
 3. Make sure the script is executable by the user running the zabbix-server
-4. If you have SELinux active, make sure that the Zabbix-Server is allowed to
-   execute the scripts (you might need to load the provided
-   [SELinux policy](../selinux/rabezbxsensordiscovery.te)). 
+4. If you have SELinux active, make sure that the Zabbix server is allowed to
+   execute the script and is allowed to operate on directories and files within
+   its home directory (this is required for `ipmi-sensor`'s cache
+   (`~/.freeipmi/).
+   On CentOS 7 the following steps were necessary:
+   ```bash
+   # Create the missing home directory of the zabbix user
+   mkdir -m 750 /var/lib/zabbix
+   chown zabbix:zabbix /var/lib/zabbix
+
+   # Set the existing SELinux user and type 
+   chcon -u system_u -t zabbix_var_lib_t /var/lib/zabbix/
```

Comment:
`chcon` changes are not persisted to the disk (ie. they get lost after a reboot). `/var/lib/zabbix/` should already be tagged as `zabbix_var_lib_t` from the system policy (I'm not sure if it's allowed from the `system_u` context though). 

Running `restorecon -Rv /var/lib/zabbix` after `mkdir`/`chown` should be enough if everything is as I expect. If that isn't enough the `system_u` context might need setting with an additional `semanage fcontext` entry (those can also be done using a file in a fashion similar to `tmpfiles.d`).

We might also consider making our Specfile own a subdir of `/var/lib/zabbix` in it's `%files` section. This could make everything work out of the box without any `mkdir`/`chown` incantations.